coalib/bearlib/languages/definitions/Matlab.py

```diff
@@ -4,5 +4,5 @@
 @Language
 class Matlab:
     aliases = 'Octave',
-    extensions = '.m'
+    extensions = ('.m',),
```

Comment:
Incorrect. See other files.

Reply:
Since file extensions seem to be similarly unformatted in other language classes, is the intent of this issue to add .mat to the `extensions` field?